src/main/java/com/minecolonies/coremod/commands/citizencommands/ListCitizensCommand.java

```diff
@@ -122,12 +159,13 @@ public void execute(@NotNull final MinecraftServer server, @NotNull final IComma
               citizen.getName())).setStyle(new Style().setClickEvent(new ClickEvent(ClickEvent.Action.RUN_COMMAND,
                                                                                      String.format(COMMAND_CITIZEN_INFO, citizen.getColony().getID(), citizen.getId())))));
 
-            citizen.getCitizenEntity().ifPresent(entityCitizen -> {
+            citizen.getCitizenEntity().ifPresent(entityCitizen ->
+            {
                 final BlockPos position = entityCitizen.getPosition();
                 sender.sendMessage(new TextComponentString(String.format(COORDINATES_XYZ, position.getX(), position.getY(), position.getZ())));
             });
         }
-        drawPageSwitcher(sender, page, citizenCount, halfPage, colonyId);
+        drawPageSwitcher(sender, page, citizenCount, halfPage, (null != colony ? colony.getID() : -1));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "true" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2583)